book/src/run_a_node.md

```diff
@@ -0,0 +1,169 @@
+# Run a Node
+
+This document provides detail for users who want to run a Lighthouse beacon node.
+You should be finished with one [Installation](./installation.md) of your choice to continue with the following steps:
+
+1. Enable [checkpoint sync](#step-1-checkpoint-sync);
+1. Set up an [execution node](#step-2-set-up-an-execution-node);
+1. Run [Lighthouse](#step-3-run-lighthouse);
+1. [Check logs](#step-4-check-logs); and
+1. [Further readings](#step-5-further-readings).
+
+Checkpoint sync is *optional*; however,  we recommend it since it is substantially faster
+than syncing from genesis while still providing all the same features.
+
+## Step 1: Choose a checkpoint sync provider
+
+Since version 2.0.0 Lighthouse supports syncing from a recent finalized checkpoint.
+The checkpoint sync can be done using [another synced beacon node](#automatic-checkpoint-sync) or a [public endpoint](#use-a-community-checkpoint-sync-endpoint) provided by the Ethereum community.
+
+In [step 3](#step-3-run-lighthouse), when running Lighthouse,
+we will enable checkpoint sync by providing the URL to the `--checkpoint-sync-url` flag.
+For now, we will copy the URL to the clipboard, selected in one of the following ways.
+
+### Automatic checkpoint sync
+
+To begin automatic checkpoint sync you will need HTTP API access to another synced beacon node.
+In this case, the URL could look like this: `http://remote-bn:5052`.
+
+> **Security Note**: You should cross-reference the `block_root` and `slot` of the loaded checkpoint
+> against a trusted source like a friend's node, or a block explorer.
```

Comment:
Let's delete this section from here and move the checkpoint sync security check to the `Check logs` section.